src/CSS/PokeCard.css

```diff
@@ -0,0 +1,10 @@
+.poke-card {
+  background-color: antiquewhite;
+  border: 1px solid red;
+  height: 20vh;
```

Comment:
I don't think you need this height or width since you're handling the sizing with grid! You might want to add a grid gap to that parent element.